.github/workflows/release.yaml

```
@@ -0,0 +1,90 @@
+name: Release
+
+on: workflow_dispatch
```

suggestion: if possible limit this workflow to run only on `main` branch

Release should never be made on other than `main` branch.

```suggestion
on:
  workflow_dispatch:
    branches:
      - main
```